lib/modules/datasource/terraform-provider/index.ts

```diff
@@ -282,6 +283,120 @@ export class TerraformProviderDatasource extends TerraformDatasource {
     return filteredResult.length === result.length ? filteredResult : null;
   }
 
+  @cache({
+    namespace: `datasource-${TerraformProviderDatasource.id}-zip-hashes`,
+    key: (registryURL: string, repository: string, version: string) =>
+      `${registryURL}/${repository}/${version}`,
+  })
+  async getZipHashes(
```

Comment:
This function shouldn't be necessary anymore if replaced with the extended Terraform build